### PR TITLE
Update Dockerfile to remove the maintainer tag

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,7 @@ RUN make build
 FROM registry.ci.openshift.org/ocp/4.14:base
 LABEL io.k8s.display-name="openshift-state-metrics" \
       io.k8s.description="This is a component that exposes metrics about OpenShift objects." \
-      io.openshift.tags="OpenShift" \
-      maintainer="Haoran Wang <haowang@redhat.com>"
```
Comment (slashpai) [inline Dockerfile:10]: May be this can be pointed to monitoring team
 
 `maintainer="OpenShift Monitoring Team <team-monitoring@redhat.com>"`

Comment (wanghaoran1988) [reply Dockerfile:10]: thanks, updated

**Review by raptorsun — APPROVED** [review]

**Pull request merged**